datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+import click
+
+from ...utils import complete_valid_checks, get_valid_integrations, read_readme_file
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+IMAGE_EXTENSIONS = {"png", "jpg"}
+NON_TILE_INTEGRATIONS = {"sortdb", "hbase_master", "kube_proxy"}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate README.md files')
+@click.pass_context
+@click.argument('integration', autocompletion=complete_valid_checks, required=False)
+def readmes(ctx, integration):
+    """Validates README files
+
+    If `check` is specified, only the check will be validated,
+    otherwise all README files in the repo will be.
+    """
+
+    repo = ctx.obj['repo_name']
+
+    errors = False
+    integrations = []
+
+    if integration:
+        integrations = [integration]
+    else:
+        integrations = sorted(get_valid_integrations())
+
+    for integration in integrations:
+        has_overview = False
+        has_setup = False
+
+        lines = read_readme_file(integration)
+        for line_no, line in lines:
+
+            if "## Overview" == line.strip():
+                has_overview = True
+
+            if "## Setup" == line.strip():
+                has_setup = True
+
+            for ext in IMAGE_EXTENSIONS:
```

Comment:
Do we have cases where the images are not hosted in the repo itself?

Reply:
IIRC, we generally encourage the images to be hosted within the repo in order to properly render for docs.